packages/webapp/components/invite/Referral.tsx

```diff
@@ -31,7 +31,7 @@ export function Referral({
   const handleAcceptClick = () => {
     // since in the page view, query params are tracked automatically,
     // we don't need to send the params here explicitly
```

Comment:
I'm still not sure what this even means, perhaps simply remove it for now.

Reply:
Because in the taxonomy, there was a mention of additional properties that are found in the query params.